visualization/src/pcl_visualizer.cpp

```diff
@@ -2137,6 +2137,7 @@ pcl::visualization::PCLVisualizer::addModelFromPolyData (
   vtkSmartPointer<vtkLODActor> actor;
   createActorFromVTKDataSet (polydata, actor);
   actor->GetProperty ()->SetRepresentationToWireframe ();
```

Comment:
I see no reason to display the mesh as a wire-frame by default, it's never done anywhere else.
Maybe we should remove this?

It does not c